package.json

```diff
@@ -16,10 +16,16 @@
   "bugs": {
     "url": "https://github.com/SU-SWS/decanter/issues"
   },
+  "dependencies": {
```

Comment:
External libraries should be managed through a package manager and never committed directly to the project repository.